### PR TITLE
OHRI-452 overall UI

### PR DESCRIPTION
**Pull request by pirupius**

### CODE REVIEW

File: src/ohri-dashboard/ohri-dashboard.meta.tsx [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { Home32, ListBulleted32, Medication32, Coronavirus32, Calendar32 } from '@carbon/icons-react';
+import { SideNavLink, SideNavMenu, SideNavMenuItem } from 'carbon-components-react';
+import { ExtensionSlot, navigate } from '@openmrs/esm-framework';
+
+export const clearDashboardSidenavRegistry = () => localStorage.removeItem('ohri-dashboard-sidenav-items');
```
Comment (samuelmale) [inline src/ohri-dashboard/ohri-dashboard.meta.tsx:6]: @pirupius where are you referencing this function?

Comment (pirupius) [reply src/ohri-dashboard/ohri-dashboard.meta.tsx:6]: @samuelmale i need to verify if we're still using it and delete if not

Comment (pirupius) [reply src/ohri-dashboard/ohri-dashboard.meta.tsx:6]: Just checked and we don't need it, i've deleted it from my working branch

Comment (samuelmale) [reply src/ohri-dashboard/ohri-dashboard.meta.tsx:6]: @pirupius do you want to remove it from this branch?

Comment (pirupius) [reply src/ohri-dashboard/ohri-dashboard.meta.tsx:6]: @samuelmale you can approve this PR, am making another push to fix some responsive css as well as deleting that

**Review by samuelmale — APPROVED** [review]

File: src/ohri-dashboard/ohri-dashboard.meta.tsx (hunk 1 above)
Comment (pirupius) [reply src/ohri-dashboard/ohri-dashboard.meta.tsx]: This has been fixed and am resolving this conversation.

**Pull request merged**